src/main/resources/api/api.yaml

```diff
@@ -748,6 +816,73 @@ definitions:
   #
   # Quizzes
   #
+  V1Quiz:
+      required:
+        - id
+        - codeExpired
```

Comment:
think about required fields, as I fink, id is not required.
Think about others required.